javascript/fluent.js

```diff
@@ -166,6 +166,10 @@
 					input.val(defaultValue);
 				} else if (input.is('textarea')) {
 					input.text(defaultValue);
+					// If it's a WYSIWYG editor, trigger it to update
+					if ('true' === input.attr('tinymce')) {
+						tinyMCE.activeEditor.setContent(defaultValue);
```

Comment:
Is it possible (for the sake of multi-editor environments) to get the correct instance?

`tinymce.get('editor');` will get an editor by ID. That way multiple editors won't end up getting all the same content. :P

http://archive.tinymce.com/wiki.php/API3:method.tinymce.get